api/models/User.js

```diff
@@ -31,4 +40,100 @@ userSchema.statics.create = function(name) {
 	});
 };
 
+userSchema.statics.read = function(GoogleID)
+{
+	return new Promise((resolve , reject) => {
+		mongoose.model('User', userSchema).findOne({googleID: GoogleID}, (err,user)=>{
+			if(err) reject(err);
+			else resolve(user);
+		});
+	});
+}
+
+userSchema.statics.delete = function(GoogleID)
+{
+	return new Promise((resolve, reject)=>{
+		mongoose.model('User', userSchema).findOne({googleID: GoogleID}).remove((err,offer)=>{
+			if(err) reject(err);
+		});
+	});
+}
+
+
+// Static: User.doSomething
+// Instance method: 
+// var user;
+// user.doSomething()
+
+
+userSchema.methods.setisAdminTo = function(becomeAdmin)
+{
+	this.isAdmin = becomeAdmin;
+	console.log('set user name:' + this.name + ' isAdmin to ' + becomeAdmin);
+	
+	return new Promise((resolve, reject)=>{
+		this.save((error, changedUser) => {
+			if(error) reject(error);
+			else resolve(changedUser);
+			});
+		});
+}
+
+
+userSchema.methods.setisMentorTo = function(becomeMentor)
```

Comment:
Name method better, maybe `setMentorStatus`? and parameter name to `mentorStatus`?